types/wx-js-sdk-dt/wx-js-sdk-dt-tests.ts

```diff
@@ -0,0 +1,7 @@
+// No npm package for wx-js-sdk, source location is: https://res.wx.qq.com/open/js/jweixin-1.2.0.js
+declare const wx: any;
```

Comment:
You shouldn't need this - this isn't importing the types you wrote. Use, in your case `/// <reference types="wx-js-sdk" />` instead, which should make a global `wx` visible. Right now your test isn't testing anything.